cmake/qtifw/install_win_copydll.qs

```diff
@@ -75,8 +85,7 @@ function Component()
       // the opposite (= Mkdir), plus it doesn't delete an empty directory anyways and we use copy (not move) above...
       // component.addElevatedOperation("Rmdir", "@TargetDir@/temp");
       // /S = recursive, /Q = quiet
-      component.addElevatedOperation("Execute", "cmd" , "/C",  "rmdir", "/S", "/Q", "@TargetDir@\\temp");
-
+      component.addElevatedOperation("Execute", "cmd", "/C", "rmdir", "/S", "/Q", tempDir);
```

Comment:
Using the new variable, and adding a local scope to the `i` loop index.  No problem.  Although it would be nice if QJSEngine had range based loops...do they?  Obviously not worth holding this up for that, just asking.

Reply:
Array.prototype.forEach works, I tested it on a testbed I have. 

```javascript
systemArray.forEach(element => {
    console.log(element);
});
```

Reply:
Well, no problem having it in here like this.  I guess if we ever touch this code again, I'll maybe remember that.  Thanks for commenting back about it.